snippets/csharp/language-reference/operators/ArithmeticOperators.cs

```diff
@@ -0,0 +1,247 @@
+using System;
+
+namespace operators
+{
+    public static class ArithmeticOperators
+    {
+        public static void Examples()
+        {
+            Console.WriteLine("=== ++ and -- operators");
```

Comment:
```suggestion
            Console.WriteLine("==== ++ and -- operators");
```

Reply:
I'm making the suggestion to change the `===` to `====` to avoid potential confusion with the JavaScript `===` operator. I doubt there would be much confusion, but it's also an easy change.